docs/organizations/managing-security-and-risk.md

```diff
@@ -514,6 +515,27 @@ Security and risk management supports checking the languages and infrastructure-
   </tbody>
 </table>
 
+## Dependencies {: id="dependencu-list"}
+
+The **Security and risk management dependencies** page displays a unified view of all dependencies used by your repositories. 
```

Comment:
should we make a note before we present the page, stating this is only available to sca users currently?